packages/rabbitmq/README.md

```diff
@@ -95,6 +96,8 @@ Import and add `RabbitMQModule` it to the `imports` array of module for which yo
 
 If you are using exchanges, provide information about them to the module and they will be automatically asserted for you as part of initialization. If you don't, it's possible message passing will fail if an exchange is addressed that hasn't been created yet.
 
+You can also optionally create your own channels which you consume messages from. If you don't create your own channels there will always be one created by default. You can also select which channel is default if you are creating your own. By setting `prefetchCount` for a particular channel you can manage message speeds of your various handlers on the same connection.
```

Comment:
Could we also mention the `default` property inside a channel definition, that would probably help when people are looking to define the default custom channel